Backpack/Toast/Classes/BPKToast.m

```diff
@@ -128,12 +146,24 @@ - (void)hudWasHidden:(MBProgressHUD *)hud {
     }
 }
 
+#pragma mark - Private methods
+
 - (void)setupHUD {
     self.hud.bezelView.style = MBProgressHUDBackgroundStyleSolidColor;
     self.hud.bezelView.backgroundColor = [[BPKColor dynamicColorWithLightVariant:BPKColor.skyGray darkVariant:BPKColor.blackTint03] colorWithAlphaComponent:0.85];
     self.hud.contentColor = BPKColor.white;
     self.hud.delegate = self;
     self.hud.accessibilityIdentifier = ToastAccessibilityIdentifier;
+    self.hud.isAccessibilityElement = NO;
+    self.hud.label.isAccessibilityElement = NO;
+    self.hud.detailsLabel.isAccessibilityElement = NO;
```

Comment:
Do we need to hide the spinner too?

Reply:
yes, but it's not publicly accessible from what I can tell

Reply:
Maybe we should recurse over the subviews then?